misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -160,6 +160,7 @@
 import io.ballerina.compiler.syntax.tree.RecordTypeDescriptorNode;
```

Comment:
Shall we implement the `RequiredExpressionNode` [1] as well in this PR? Seems to be a new node.

[1] https://github.com/ballerina-platform/ballerina-lang/blob/f919b1d55a84b1d302f188e657cf5155755c0327/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/TreeModifier.java#L3157

Reply:
Implemented.